main.tf

```diff
@@ -513,7 +511,7 @@ resource "aws_iam_role_policy_attachment" "ssm" {
   count = var.enable_manage_gitlab_token ? 1 : 0
```

Comment:
Here the variable is still used. Evoluation will now cause an error, since the value is always null.